components/Registration/Registration.tsx

```diff
@@ -38,13 +40,16 @@ export const Registration = ({
   formID,
   secondLogo,
   dark = false,
+  previewImage,
+  imageContainerHeight = [158, 300],
 }: RegistrationProps) => {
   return (
     <Pattern1
       headTitle={headTitle}
       headDescription={headDescription}
       colorStyle="purple"
       secondLogo={secondLogo}
+      previewImage={previewImage ? previewImage : null}
```

Comment:
This can be:
```suggestion
      previewImage={previewImage}
```
It will be undefined if nothing is passed in and trigger the default for `<Pattern1/>`

Reply:
That's true, updating..

Reply:
It also looks like null results in an unintended og_image value:
<img width="1433" alt="Screen Shot 2022-05-04 at 2 37 04 PM" src="https://user-images.githubusercontent.com/70108137/166803284-a12d5129-81a0-4970-9ff9-7acba0bf8bf9.png">

Reply:
updated